extensions/geode-modules-tomcat8/src/distributedTest/java/org/apache/geode/modules/session/Tomcat8SessionsDUnitTest.java

```diff
@@ -53,6 +53,7 @@ public void setUp() throws Exception {
 
     sessionManager.getTheContext().setSessionTimeout(30);
     region.clear();
+    checkSanity();
```

Comment:
Did you mean to leave `checkSanity()` in the code?

Reply:
Yes.